README.md

```diff
@@ -50,6 +50,21 @@ api = onfido.Api("<YOUR_API_TOKEN>", base_url=Region.CA)
 
 See https://documentation.onfido.com/#regions for supported regions.
 
+### Timeouts
+
+You can optionally set a global timeout for all requests in the API
+constructor. This takes a floating number input and each whole integer
+increment corresponds to a second. 
+
+For example, to set a timeout of 1 second:
+
+```python
+api = onfido.Api("<YOUR_API_TOKEN>", timeout=1)
+```
+
+The default value for `timeout` is `None`, meaning no timeout will be set on
+the client side.
```

Comment:
Are we certain about this? Would be good to understand if requests library has any default timeout.

Reply:
From their docs:

> Most requests to external servers should have a timeout attached, in case the server is not responding in a timely manner. By default, requests do not time out unless a timeout value is set explicitly. Without a timeout, your code may hang for minutes or more.

I think I'm actually setting it to `None` by default here in any case?